Content.Client/ClientNotifyManager.cs

```diff
@@ -61,7 +61,7 @@ public void PopupMessage(ScreenCoordinates coordinates, string message)
             var label = new PopupLabel {Text = message};
             var minimumSize = label.CombinedMinimumSize;
             label.InitialPos = label.Position = coordinates.Position - minimumSize / 2;
-            _userInterfaceManager.StateRoot.AddChild(label);
```

Comment:
Yeah I'd rather this not be made part of WindowRoot. Should make a new control after window root and use that instead.